package.json

```diff
@@ -175,6 +175,21 @@
         "command": "qute.validation.enabled.toggle.on",
         "title": "Enable Qute Validation",
         "icon": "$(eye-closed)"
+      },
+      {
+        "command": "qute.refactor.surround.with.comments",
+        "when": "quarkusProjectExistsOrLightWeight && editorLangIdSupportsQute && QuteLSReady",
+        "title": "Qute: Surround with Comments"
+      },
+      {
+        "command": "qute.refactor.surround.with.cdata",
+        "when": "quarkusProjectExistsOrLightWeight && editorLangIdSupportsQute && QuteLSReady",
+        "title": "Qute: Surround with Unparsed Character Data"
+      },
+      {
+        "command": "qute.refactor.surround.with.section",
+        "when": "quarkusProjectExistsOrLightWeight && editorLangIdSupportsQute && QuteLSReady",
```

Comment:
where you defineeditorLangIdSupportsQute? I suggest that you do like vscode-xml by writing `editorLangId in qute.supportedLanguageIds` and you fill  `qute.supportedLanguageIds` like https://github.com/redhat-developer/vscode-xml/blob/a17945188ade7f39a3d8ee0fbc75443db5ac3200/src/extension.ts#L46

Reply:
It's defined here: https://github.com/redhat-developer/vscode-quarkus/blob/2f275047459fe3791874a29e92c8657e4e9f8f98/src/qute/commands/registerCommands.ts#L210
and it's used to display the validation icon for Qute templates